skills/vendure-cli/commands/dev.md

```
@@ -0,0 +1,49 @@
+# vendure dev
+
+Runs Vendure in development mode. By default it starts three processes: the
+GraphQL **server** (`ts-node ./src/index.ts`), the **worker**
+(`ts-node ./src/index-worker.ts`), and the **dashboard** (a Vite dev server).
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)